MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Reflectometry/ReflCommandAdapter.h

```diff
@@ -73,8 +76,11 @@ public slots:
     connect(action, SIGNAL(triggered()), this, SLOT(call()));
   };
   // The adaptee
-  ReflCommandBase_sptr m_adaptee;
+  ReflCommand_sptr m_adaptee;
+  std::vector<std::unique_ptr<ReflCommandAdapter>> m_adapter;
 };
+
+typedef std::unique_ptr<ReflCommandAdapter> ReflCommandAdapter_uptr;
```

Comment:
c++11 has a nice feature called `using` to replace typedefs: You could re-write this as:

```cpp
using ReflCommandAdapter_uptr = std::unique_ptr<ReflCommandAdapter>;
```